### PR TITLE
add date modified and size

### LINKED ISSUE #1: Add date modified size and maybe type to file view

### PR DESCRIPTION
**Pull request by yashdavisgupta**
resolves #1

### CODE REVIEW

File: cli.js [hunk 1]
```diff
@@ -41,8 +41,6 @@ app.get('/b', function(req, res) {
 app.use(express.static(__dirname)); // module directory
 var server = http.createServer(app);
 
-asp.setcwd(dir, argv.include, argv.exclude);
```
Comment (AlexanderOtavka) [inline cli.js:44]: This looks like a mistake.  I think it's causing this error on startup:
```
web_1  | TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string or an instance of Buffer or URL. Received undefined
web_1  |     at Object.readdir (fs.js:996:10)
web_1  |     at readRoots (/usr/app/lib/index.js:59:8)
web_1  |     at exports.get (/usr/app/lib/index.js:86:5)
web_1  |     at Layer.handle [as handle_request] (/usr/app/node_modules/express/lib/router/layer.js:95:5)
web_1  |     at next (/usr/app/node_modules/express/lib/router/route.js:137:13)
web_1  |     at Route.dispatch (/usr/app/node_modules/express/lib/router/route.js:112:3)
web_1  |     at Layer.handle [as handle_request] (/usr/app/node_modules/express/lib/router/layer.js:95:5)
web_1  |     at /usr/app/node_modules/express/lib/router/index.js:281:22
web_1  |     at Function.process_params (/usr/app/node_modules/express/lib/router/index.js:335:12)
web_1  |     at next (/usr/app/node_modules/express/lib/router/index.js:275:10)
```

**Review by AlexanderOtavka — CHANGES_REQUESTED** [review]
I can't finish the review until you fix the error, but the code seems fine

**Comment by yashdavisgupta** [comment]
Should work now

File: app/app.js [hunk 2]
```diff
@@ -56,10 +56,9 @@ function constructTable(data){
   tab = document.createElement('table');
   for(file in data){
     tr = document.createElement('tr');
-    console.log(data[file].Path)
     if(data[file].IsDirectory){
       path = '/files?path='+data[file].Path
-      tr.setAttribute("onclick","refreshTable('"+path+"'); currentPath = path;");
+      tr.setAttribute("onclick","refreshTable('"+path+"'); currentPath = '"+data[file].Path+"';");
```
Comment (YourFin) [inline app/app.js:61]: I do not envy writing code like this

**Review by YourFin — APPROVED** [review]
Looks OK to me

**Review by YourFin — APPROVED** [review]
Looks OK to me

**Pull request merged**